bucket/temurin21-jdk.json

```diff
@@ -0,0 +1,40 @@
+{
+    "description": "Eclipse Temurin is a runtime provided by Eclipse Adoptium for general use across the Java ecosystem",
+    "homepage": "https://adoptium.net",
+    "version": "21.0.0-35.0.LTS",
+    "license": "GPL-2.0-only WITH Classpath-exception-2.0",
+    "architecture": {
```

Comment:
Just a heads up that 21.0.1 is being cooked right now [1], should be published soon (:crossed_fingers: ), thank you

[1] https://github.com/adoptium/temurin/issues/6